src/platform/backends/CMakeLists.txt

```diff
@@ -14,6 +14,6 @@
 #
 # Authored by: Alberto Aguirre <alberto.aguirre@canonical.com>
 
-add_subdirectory(backend_utils)
+add_subdirectory(shared_linux)
```

Comment:
As discussed yesterday, I think that limits it too much. Most of this is cross-platform, some of it unix-specific, but I don't think there's anything Linux-specific in this PR yet?

Reply:
I did look and notice that none of our non-linux code depends on the contents of the directory. Hence the name!

What we discussed did make sense, but I was wary of making too big an MP by breaking up this directory into shared, shared/linux etc. But I'll do so now if you like.

Reply:
Yeah I'd say that makes sense.

Reply:
I'm struggling with this change, as while now there isn't anything linux specific in the Process* bits, the next MP will introduce apparmor, which does make them linux specific.

So I'd be moving things around, just to move them back to where they are now.

(Backend_utils is more than 50% linux-only code too)

I agree with you that Process would be great as a shared thing, and likely there'll be more duplicate code we can combine. I'll definitely do that. But I don't think I've headspace to tackle that problem and add the apparmor bits simultaneously. I'd rather add the apparmor bits first, then identify the platform commonalities after.

So can I request a "let me do it later?"